client/app/AddressList.jsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import Address from './Address';
+import AddressCard from './AddressCard';
+
+class AddressList extends React.Component {
+    render() {
+        const{contacts} = this.props
+        const arrContacts = Object.values(contacts)
+        return(
+            arrContacts.map((contact) => {
```

Comment:
change this into a ul with li elements containing the address card, also provide keys using contact.id